docs/topics/multiplatform/multiplatform-expect-actual.md

```diff
@@ -399,6 +399,24 @@ typically provides an expected `CommonViewModel` class whose actual Android coun
 from the Android framework. See [Use platform-specific APIs](multiplatform-connect-to-apis.md#adapting-to-an-existing-hierarchy-using-expected-actual-classes)
 for a detailed description of this example. -->
 
+#### Limitations of open expected classes
```

Comment:
I don't know if it's common in our documentation, but "expected" feels wrong to me.

`expect` is a keyword and converting it to the adjective sounds incorrect

Reply:
I don't disagree, and I avoided using the adjective in the text. But writing "expect classes" in the title, when we cannot highlight the keyword, reads like a grammar mistake. All the rest of the modifiers I can remember are adjectives to begin with (internal, abstract, actual...), so they don't have this problem. Here, we need a compromise.

Right now the concept we introduce on the page is "expected declarations," and that's the convention. I think it's possible to have clearer terminology here. We'll keep thinking about this.